CHANGELOG.md

```diff
@@ -3,6 +3,177 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.34](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.33...v4.0.0-alpha.34)
+
+_July 21, 2021_
+
+Big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
```

Comment:
The most important change of this release is IMHO that we fix v4 support. Could we mention it?

Reply:
Sure